### ISSUE TITLE
Coronavirus Dashboard 

### ISSUE THREAD

**Issue by MauroCapolino** [CSSEGISandData/COVID-19#1223]
Hello I created based on your data and the Italian data a dashboard that I update daily with over 50 analysis. I also create automatically a pdf with all the reports. Happy to help you in case you want to use part of it
The dashboard is on google data studio 
https://datastudio.google.com/u/0/reporting/9ce371f1-62fa-4f2e-8d3c-da4d6dff967b

**Comment by JiPiBi** [comment]
Seems great but, I get this error when linking :  
Erreur de configuration de l'ensemble de données

> Impossible d'accéder à votre ensemble de données depuis Data Studio. 

The only issue could be  the global reliability of data here  (for Italian values I have no doubt)

**Comment by MauroCapolino** [comment]
The global data is under update since I had to change my dB this morning to adapt to tew John Hopkins format
Will be online in 1 hr
Let me know email if you like to get pdf

Mauro

**Comment by joaopedrosantos** [comment]
interesting, will look at google data studio. any tips to learn?

**Comment by JiPiBi** [comment]
@MauroCapolino 
Now I have access 
But I dont understand your calculations for each country  
I understand that you give the daily value for the 2 last days , but on which basis are the percentages calculated  : from one day vs the previous ? 

And mainly what represent the total (as the values are cumulative why have a total ??)
and for the percentage on the line total general ,  on what is it based .

For Netherlands  363900%  and United Kingdom 0%  are quite strane too ?? 

The same for the deaths ....

**Comment by MauroCapolino** [comment]
Yes incremental is vs previous day
Yesterday I did major review and fix
Check again

Mauro

**Comment by MauroCapolino** [comment]
Just open an account put some data in a google sheets and then from studi link it to the sheets the rest is basic reporting 

Mauro

**Comment by JiPiBi** [comment]
In first tab , you are adding all the time series values for each  country which en fact  are cumulative  and not daily values  : so it needs to be fixed

**Comment by MauroCapolino** [comment]
Please send me the page name so that I can check I have 57 pages

Mauro

**Comment by JiPiBi** [comment]
As I said it is the first , look for example for France on the third value in first col , I repeat that you add cumulative values from the beginning , instead of keeping the last one  , I summed up  to confirm your value.

**Comment by MauroCapolino** [comment]
I do not know what is the issue

Nella mia pagina iniziale i casi totali della Francia sono 20123, Italy 63927...

Che e’ lo stesso dato del sito Johm Hopkins totale casi 

Non vedo il problema!


On 24 Mar 2020, at 06:15, JiPiBi <notifications@github.com> wrote:


As I said it is the first , look for example for France on the third value in first col , I repeat that you add cumulative values from the beginning , instead of keeping the last one , I summed up to confirm your value.

**Comment by JiPiBi** [comment]
La mia famiglia e venuta nella Francia da San Giovanni Bianco 
Parlo un po l'italiano, ma non troppo e con molta falta :-), as my english in fact   :-( 

Your dashboard has changed now , so it seems you fixed the issue 

![image](https://user-images.githubusercontent.com/44967630/77416691-7c9e7b00-6dc4-11ea-8e71-68f41643421b.png)

In my last check,  for France the third line was labelled **Total gen** and was the sum of all cumulative value . 

I will go on explore your numerous pages 
Is there a mean to send you my remarks not in this github repo (directly on your site ?)

**Comment by MauroCapolino** [comment]
Yes I did major fix work and now there is a mobile version at the link in the orange part try it


On 24 Mar 2020, at 11:49, JiPiBi <notifications@github.com> wrote:


La mia famiglia e venuta nella Francia da San Giovanni Bianco
Parlo un po l'italiano, ma non troppo e con molta falta :-), as my english in fact :-(

Your dashboard has changed now , so it seems you fixed the issue

 <https://user-images.githubusercontent.com/44967630/77416691-7c9e7b00-6dc4-11ea-8e71-68f41643421b.png>
In my last check, for France the third line was labelled Total gen and was the sum of all cumulative value .

I will go on explore your numerous pages
Is there a mean to send you my remarks not in this github repo (directly on your site ?)

**Comment by JiPiBi** [comment]
In page 4 , what disturbs me is that I think that France and Italy are following the same curve (the slopes seem the same)
but I can't really compare them because the beginning of epidemics  is in hard code

At the beginning the values are small , and the true date of the beginning is not easy to choose, you could also say that the first cases in France were earlier , why not ( in fact it was end of january)

So can I modify myself this beginning date of a country  to compare the trends  ?

**Comment by JiPiBi** [comment]
A suggestion : Is it possible to go directly to a given page,  because there are a lot and you have your favourite ones (by a dropdown list ?)

**Comment by MauroCapolino** [comment]
Changed the menu to show all the pages on the left( you can close the list of page with the arrow on top right of the list)
Let me know if you like it


Mauro

**Comment by MauroCapolino** [comment]
The model is one so if I allow you to change it would change for all the users 

Sorry the tool does not allow per user parameters input. If I add a linked field to the model with the start date and you change it than all can change it causing conflict 

I anyhow do not consider relevant the first 10 relative days since all the number are low what is important is how each country accelerates like USA


Mauro

**Comment by JiPiBi** [comment]
Sorry for my late answer , I was struggling with new detailled data for France that give at a departement level hospitalized and ICU (data you already have for Italy at a region level )

For me now what is important is to know hosp + ICU for all the countries : I wrote an issue on that #1677 
I hope you share my point of view :-)

About your dashboard , the improvment with left menu is great !

I'm trying now to learn DataStudio to publish data , because python is great for anlysis and cleaning of data , but views are poorly illustrated

**Comment by JiPiBi** [comment]
Your  charts are based on a given date that is supposed to be the start of the epidemic in a country.
Dont you think that it will be interesting to measure the confinment impact ? 

Another question :
For me  the new hospitalized and the new ICU for a given day  is an important data 
In France as data are given mainly for hospitals I think that I can measure the new hospitalized people , more difficult for ICU because people in ICU can deceased or stay in hospital , so the movments  or flows are not clear to me ( I can suppose that deceased came from ICU but with the difficult choices they have to make , not completely sure )
What do you think of that KPI ?
